config/crd/bases/services.platform.opendatahub.io_monitorings.yaml

```diff
@@ -52,6 +52,9 @@ spec:
           spec:
             description: MonitoringSpec defines the desired state of Monitoring
             properties:
+              metrics:
+                description: monitoring spec exposed only to internal api
```

Comment:
I don't think this is true, and the description should describe what it does which would be more along the lines of 

"Enables metrics collection"